h/services/mention.py

```diff
@@ -3,13 +3,12 @@
 
 from sqlalchemy import delete
 from sqlalchemy.orm import Session
+from util.markdown_render import MENTION_ATTRIBUTE, MENTION_USERID
```

Comment:
It makes more sense that a domain-specific service depends on an external utility than the other way around, so I moved these constants there and imported them here.